File: findmps/getmps.py
import sys

from makesqlitedb import MP, Policy, Distance, sqlite3, dbpath

try:
    query = sys.argv[1]
except:
    print "Usage: %s keyword" % sys.argv[0]
    sys.exit(1)

conn = sqlite3.connect(dbpath)
c = conn.cursor()

#sql = "SELECT distance.id,distance.distance from distance WHERE distance.dream_id IN ('1034')"
#sql = "SELECT dream_id from policy WHERE UPPER(policy.title) LIKE UPPER('%%%s%%')" % (query)
#sql = "SELECT distance.mp_id from distance WHERE distance.dream_id IN (SELECT dream_id from policy WHERE UPPER(policy.title) LIKE UPPER('%%%s%%'))" % (query)
#sql = "SELECT distance.mp_id, distance.distance from distance WHERE distance.dream_id IN (SELECT dream_id from policy WHERE UPPER(policy.title) LIKE UPPER('%%%s%%')) ORDER BY distance.distance LIMIT 10" % (query)
sql = "SELECT DISTINCT mp.mp_id,mp.first_name,mp.last_name FROM mp WHERE mp.mp_id IN (SELECT DISTINCT distance.mp_id from distance WHERE distance.dream_id IN (SELECT dream_id from policy WHERE UPPER(policy.title) LIKE UPPER('%%%s%%')) ORDER BY distance.distance LIMIT 10)" % (query)
for i in c.execute(sql):
    print i

#for query in queries:
#
#    searchTerm = "UPPER(%s) LIKE UPPER('%%%s%%')" % ('title', query)
#    for i in Policy.select(searchTerm):
#        print "Policy:", i
#        print "Distances:", i.distances
#        #print "MPs:", i.mps.count()
#        #for j in i.mps:
#        #    print j
#        #print
#
#for mp in MP.select():
#    print mp

File: scripts/getmps.php
<?php
	session_start();
	$dbpath = ($_SERVER['DOCUMENT_ROOT'].'/mps.db');
    echo $dbpath;
    echo "\nDatabase: ";
    $db = new SQLiteDatabase('filename');
	
	if(!checkSession()){
		echo '3:You need to be logged in the create a cause:Create';
		exit;
	}

	$keyword = $_POST['keyword'];
	$keyword = $_GET['keyword'];
	$userid = getCurrentUserInfo('id');

	if($keyword==''){
		echo 'Missing keyword';
		exit;
	}

	$date = date("d-m-Y");
	$slug = rand(10000,99999);

    echo "blah";

	$keyword = mysql_real_escape_string($keyword);
    #$sql = "SELECT DISTINCT mp.mp_id,mp.first_name,mp.last_name FROM mp WHERE mp.mp_id IN (SELECT DISTINCT distance.mp_id from distance WHERE distance.dream_id IN (SELECT dream_id from policy WHERE UPPER(policy.title) LIKE UPPER('%".$keyword."%')) ORDER BY distance.distance LIMIT 10)";
    $sql = "SELECT DISTINCT mp.mp_id,mp.first_name,mp.last_name FROM mp WHERE mp.mp_id IN (SELECT DISTINCT distance.mp_id from distance WHERE distance.dream_id IN (SELECT dream_id from policy WHERE UPPER(policy.title) LIKE UPPER('%".$keyword."%')) ORDER BY distance.distance LIMIT 10)";
	$results = mysql_query($sql);
    if (!$result) {
        echo "Invalid query: ".mysql_error();
    }
    echo $results;
    echo "blah";
	exit;
?>
